app/graphql/client.tsx

```diff
@@ -95,12 +95,24 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
         }`,
       )
 
+      const wsLinkAuth = async () => {
+        if (token) {
+          return {
+            Authorization: getAuthorizationHeader(token),
+          }
+        } else if (appConfig.isAuthenticatedWithDeviceAccount) {
+          const appCheckToken = await getAppCheckToken()
+          return {
+            Authorization: appCheckToken ? getAuthorizationHeader(appCheckToken) : "",
```

Comment:
```suggestion
            Authorization: appCheckToken ? getAuthorizationHeader(appCheckToken) : undefined,
```

shouldn't it be undefined to be consistent with the return below line 109?